dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.jakarta.jaxws-3.0.feature

```diff
@@ -1,7 +1,8 @@
 -include= ~${workspace}/cnf/resources/bnd/feature.props
 symbolicName=io.openliberty.jakarta.jaxws-3.0
 singleton=true
--features=io.openliberty.jakarta.jaxb-3.0; apiJar=false
+-features=io.openliberty.jakarta.jaxb-3.0; apiJar=false, \
```

Comment:
> Should com.ibm.websphere.appserver.internal.optional.jaxws-2.2 also be updated to have eeCompatible tolerates? Would be good to update other features to use that feature possibly instead of pulling in the bundle unconditionally. Can't remember if we wanted to pull it in unconditionally in some cases. May want to have an API feature for the cases we want to pull it in unconditionally.

Probably.  Another candidate for the separate issue.